pytest_socket.py

```diff
@@ -170,16 +170,52 @@ def host_from_connect_args(args):
         return host_from_address(address)
 
 
+def is_ipaddress(address: str):
+    """
+    Determine if the address is a valid IPv4 address.
+    """
+    try:
+        socket.inet_aton(address)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏what about ipv6 addresses? There's a different function that can validate both. Docstring needs an update.

Reply:
I've changed to using [ipaddress](https://docs.python.org/3/library/ipaddress.html#ipaddress.ip_address)